noria-server/src/controller/inner.rs

```diff
@@ -720,10 +720,10 @@ impl ControllerInner {
     }
 
     /// Obtain a `ViewBuilder` that can be sent to a client and then used to query a given
-    /// (already maintained) reader node. If the view has replicas, a `ViewBuilder` is returned
-    /// for each view in round robin order.
+    /// (already maintained) reader node. If there are multiple readers for any given query,
+    /// a `ViewBuilder` is returned for each reader in round robin order.
     ///
-    /// The name of the reader node is `name`, or if it is a replica, `name_r[REPLICA_NUMBER]`.
+    /// `name` is the name of the query.
```

Comment:
Isn't name the name of the `view` that the `Reader` should be constructed for?

Reply:
What's the difference between queries and external views?

Reply:
Everything SQL statement the system is given is a query. We only add `Reader`s for those queries that the application explicitly chooses to make "public" using `CREATE VIEW` (which in turn calls `maintain`)

Reply:
Side-note: there are two other ways to get "public" queries with a reader -- using `VIEW <name>: SELECT ...` and `QUERY <name>: SELECT ...` in the recipe. Both end up calling the same code paths (`maintain`, etc.), though.